v22.1/explain-analyze.md

```diff
@@ -212,6 +212,51 @@ For example, the following `EXPLAIN ANALYZE` statement executes a simple query a
 (30 rows)
 ~~~
 
+If you perform a join, the estimated max memory allocation is also reported for the join. For example:
```

Comment:
Perhaps call out the `maximum memory usage` field and value in the following example? Or maybe just the value, if we think the field names might be subject to change.

Reply:
The example is demonstrating that `estimated max memory allocated` was added to the `lookup join` component of the `EXPLAIN ANALYZE` output.